flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/dataformat/vector/heap/AbstractHeapVector.java

```diff
@@ -109,18 +107,7 @@ public static AbstractHeapVector createHeapColumn(LogicalType fieldType, int max
 			case TIME_WITHOUT_TIME_ZONE:
 				return new HeapIntVector(maxRows);
 			case BIGINT:
-			case TIMESTAMP_WITHOUT_TIME_ZONE:
```

Comment:
looks like both `allocateHeapVectors` and `createHeapColumn` are useless?

Reply:
Uses should use `allocateHeapVectors` or `createHeapColumn` to create HeapVectors. See the java doc of constructor of `HeapIntVector`. Uses should not call `new HeapIntVector(len)` except for testing purposes.

Reply:
What kind of user you are talking about?

Reply:
May be some connectors? The classes in `org.apache.flink.table.dataformat.vector.heap` is not used except testing now, but may be used to develop some connectors? Or if the aim of these classes is only for testing, I could move then to `test` and remove the two functions.
What do you think?